bokehjs/src/lib/models/glyphs/circle.ts

```diff
@@ -282,11 +284,12 @@ export class Circle extends XYGlyph {
 
     this.mixins<Circle.Mixins>([LineVector, FillVector])
 
-    this.define<Circle.Props>(({}) => ({
+    this.define<Circle.Props>(({Number}) => ({
```

Comment:
@mattpap is this correct? `p.Number` has a deprecation comment and I don't know what your intentions are

Reply:
This is correct. This is the new approach.